src/shared/utils/now.ts

```diff
@@ -0,0 +1,3 @@
+import { formatISO } from 'date-fns';
```

Comment:
giving that we didn't use much of the date functions so far. please use dayjs to be aligned with the frontend (they also decided to use dayjs and it is quite good too)

Reply:
dayjs is now used instead of date-fns